homeassistant/components/deconz/device_automation.py

```diff
@@ -0,0 +1,120 @@
+"""Provides device automations for deconz events."""
+import voluptuous as vol
+
+import homeassistant.components.automation.event as event
+from homeassistant.const import (
+    CONF_DEVICE_ID,
+    CONF_DOMAIN,
+    CONF_EVENT,
+    CONF_PLATFORM,
+    CONF_TYPE,
+)
+
+from . import DOMAIN
+from .gateway import get_gateway_from_config_entry
+
+
+# mypy: allow-untyped-defs, no-check-untyped-defs
+
+CONF_TURN_ON = "turn_on"
```

Comment:
We should change these types. They should be `remote_button_turn_on_pressed` etc. Since Deconz is going to supply triggers for a lot of different device types.

Reply:
I think it should be split: the action/type is `pressed` and the name/event/key is `turn_on`. Seeing as the buttons can also be "long pressed" and "released".

Reply:
How would we simplify this?

Reply:
So are we going to do this with subtype now ?

Reply:
This is all subtypes now, following emontnemeries PR and the discussion in backend